src/types.ts

```diff
@@ -59,10 +69,24 @@ export interface FiatConnectApiClient {
   getCookies(): Promise<string>
 }
 
+export interface SiweClientConfig {
+  accountAddress: string
+  statement: string
+  version: string
+  chainId: number
+  sessionDurationMs: number
+  loginUrl: string
+  clockUrl: string
+}
+
 export interface LoginParams {
   issuedAt?: Date
 }
 
+export type SiweLoginParams = LoginParams & {
+  headers?: Record<string, string>
```

Comment:
What kind of header do we expect to add?

Reply:
The auth header with the client api key: https://github.com/fiatconnect/fiatconnect-sdk/blob/main/src/fiat-connect-client.ts#L62-L66